src/Pandell.PackageRunner/Program.cs

```diff
@@ -263,5 +266,12 @@ private static Parameters ParseArguments(IEnumerable<string> args)
 
             return parameters;
         }
+
+        /// <summary>
+        /// </summary>
+        private static byte[] HexStringToByteArray(string hex)
+        {
+            return Enumerable.Range(0, hex.Length/2).Select(x => Convert.ToByte(hex.Substring(x*2, 2), 16)).ToArray();
```

Comment:
I think this function is a nice candidate for a simple unit test (though it will have to be marked as at least `internal` + add `InternalsVisibleTo` attribute).